compiler/rustc_session/src/options.rs

```diff
@@ -2331,8 +2331,6 @@ options! {
         "run all passes except codegen; no output"),
     no_generate_arange_section: bool = (false, parse_no_value, [TRACKED],
         "omit DWARF address ranges that give faster lookups"),
-    no_implied_bounds_compat: bool = (false, parse_bool, [TRACKED],
-        "disable the compatibility version of the `implied_bounds_ty` query"),
```

Comment:
why remove that? isn't it still useful to test bevy itself?

Reply:
Guess so, was just kinda annoying to have flag checks everywhere. I'll put it back.